README.md

```diff
@@ -1,2 +1,127 @@
-# nf-synindex
-Nextflow pipeline for indexing S3 objects in Synapse
+# nf-synindex: Index S3 Objects in Synapse
+
+## Purpose
+
+The purpose of this Nextflow workflow is to automate the process of indexing S3 objects in Synapse. These S3 objects are typically the output files from a general-purpose (_e.g._ nf-core) workflow that doesn't contain Synapse-specific steps for uploading or indexing outputs. This workflow is intended to be run after other data processing workflows.
+
+The benefits of using this workflows include:
+
+- There is a lack of data transfer and duplication because the S3 objects remain where they are.
+- The folder-like structure in S3 is reproduced in Synapse to maintain the file organization.
+
+Briefly, `nf-synindex` achieves this automation as follows:
+
+1. Update the `owner.txt` file with the authenticated user ID
```

Comment:
Is this an issue if there are different people running this pipeline.  Did we want the `owner.txt` changing?

Reply:
The pipeline will append any missing user IDs and thus shouldn't affect the access of others to the bucket. But maybe I'm misunderstanding your concern.

Reply:
Got it, i was worried it would replace the user IDs instead of appending.